### PR TITLE
Pdi 86 eef column on ilr line

### PR DESCRIPTION
**Pull request by kuzinator**

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/LearnerMutators/FM36Base.cs [hunk 1]
```diff
@@ -122,6 +107,27 @@ protected void MutateCommon(MessageLearner learner, Learner learnerRequest)
...
+                ESMType = EmploymentStatusMonitoringType.SEM.ToString(),
+                ESMCode = (int) EmploymentStatusMonitoringCode.SmallEmployer,
+                ESMCodeSpecified = true
+            });
+
+            learner.LearnerEmploymentStatus[0].EmploymentStatusMonitoring = statusMonitorings.ToArray();
+
+            var eefCode = (LearnDelFAMCode) learnerRequest.EefCode.GetValueOrDefault((int) LearnDelFAMCode.EEF_Apprenticeship_19);
+            DCT.TestDataGenerator.Helpers.AddLearningDeliveryFAM(learner, LearnDelFAMType.EEF, eefCode);
+        }
```
Comment (CarlHA) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/LearnerMutators/FM36Base.cs]: Does this mean we would be using magic numbers in the feature file for EefCode? Would it not be clearer to use EEF_Apprenticeship_19 etc in the feature file?

Comment (kuzinator) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/LearnerMutators/FM36Base.cs]: As Rohan explained, we'd rather have a number here

Comment (kuzinator) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/LearnerMutators/FM36Base.cs]: As Rohan explained, we'd rather have a number here

Comment (CarlHA) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/LearnerMutators/FM36Base.cs]: Resolved

**Comment by CarlHA** [comment]
Replaced by #457

**Pull request closed without merging**